imod/tests/test_mf6/test_mf6_riv.py

```diff
@@ -127,6 +127,18 @@ def test_inconsistent_nan(riv_data, dis_data):
 
     assert len(errors) == 1
 
+@parametrize_with_cases("riv_data,dis_data", cases=RivDisCases)
+def test_layer_coord_allowed(riv_data, dis_data):
+    # Test if no bugs like https://github.com/Deltares/imod-python/issues/830
```

Comment:
can you expand the test a little bit with a scenario where validation should fail?

Reply:
Good call, added a separate test for that